runtime/expr_stmt.cc

```diff
@@ -344,6 +344,18 @@ interval_expr& interval_expr::operator&=(interval_expr r) {
   return *this;
 }
 
+expr interval_expr::contains(expr_ref x) const {
```

Comment:
Super nit: I'd move this up to line 242, next to the other similar helpers like `empty`.